s3/charmcraft.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+type: charm
+platforms:
+  ubuntu@22.04:amd64:
```

Comment:
Might be best to switch to Noble from the get go, since it's a major version change.

Reply:
Good idea, done in 261d6779c1d0723e5335dd900879f5f689127595